aiorwlock/__init__.py

```diff
@@ -17,15 +17,31 @@
 # implementation based on:
 # http://bugs.python.org/issue8800
 
+_global_lock = threading.Lock()
+
+
+class _LoopBoundMixin:
```

Comment:
We probably don't need a class for this, if it's only going to be used once.

Reply:
Done in 76c21a551cc30cccba21f73525dceef20766594a.